### PR TITLE
Tidy up repo structure

### PR DESCRIPTION
**Pull request by tonyshumlh**
Highlights of the changes
- Remove the redundant files
- Create `/results` and `/scripts` for the analysis batch run scripts and results storage

To-Do:
- Organize `/docs` to cater for 1) Github page to display both Proposal and Final Report, 2) ReadTheDocs document of FIXML package (if necessary)

### CODE REVIEW

File: results/figures/.gitkeep
Comment (SoloSynth1) [inline results/figures/.gitkeep:1]: @tonyshumlh Any reason(s) for keeping these empty folders?

**Review by SoloSynth1 — COMMENTED** [review]
@tonyshumlh LGTM in general! But I just have a small question on the empty folders. Are those needed (like, this is where we will put the report in?)

File: results/figures/.gitkeep
Comment (JohnShiuMK) [reply results/figures/.gitkeep:1]: @tonyshumlh @SoloSynth1 I understand that the folder `/results` and `/scripts` are potentially for the accuracy / consistency evaluation of our system. I'm thinking whether we should merge them into one single folder called `/analysis` (or some name like that)?

Comment (tonyshumlh) [reply results/figures/.gitkeep:1]: @JohnShiuMK As discussed, we can do it this way (`/analysis`) and ask for Tiffany and Simon comment on Fri

**Comment by tonyshumlh** [comment]
@JohnShiuMK @SoloSynth1 please help review and merge if it is good to go. Thanks

File: results/figures/.gitkeep
Comment (SoloSynth1) [reply results/figures/.gitkeep:1]: Still, after renaming the empty folders are still there. Any reason(s) for keeping these empty folders?

**Review by jinyz8888 — APPROVED** [review]
Looks okay

File: results/figures/.gitkeep
Comment (jinyz8888) [reply results/figures/.gitkeep:1]: empty folders are gone

**Pull request merged**